src/nvim/lua/vim.lua

```diff
@@ -38,8 +38,8 @@
 -- Returns:
 --    {errcode}, {output}
 local function _system(cmd)
-  local out = vim.api.nvim_call_function('system', { cmd })
-  local err = vim.api.nvim_get_vvar('shell_error')
+  local out = vim.api.nvim_call_function('system', { cmd }) --luacheck: ignore 113
```

Comment:
can these be removed, now that the luacheckrc is added?

Reply:
Yes.

Reply:
I just removed them.